fire-hose-management-frontend/src/components/Departments/EditDepartmentOverlay.jsx

```diff
@@ -0,0 +1,179 @@
+import {makeStyles} from "@material-ui/core/styles";
+import React, {useState} from "react";
+import {Button, Checkbox, FormControlLabel, Grid, MenuItem} from "@material-ui/core";
+import Stack from "@mui/material/Stack";
+import {Select, TextField} from "@mui/material";
+import {CSelect} from "../shared/input/CSelect";
+import {Overlay} from "../shared/Overlay";
+import {ValidatorForm} from "react-material-ui-form-validator";
+import {putDepartment} from "../../utils/requests/Departments";
+import {useMutation} from "react-query";
+import Notifications from "../shared/Notifications";
+import {CTextField} from "../shared/input/CTextField";
+
+const useStyles = makeStyles((theme) => ({
+    checkbox: {
+        display: "table-cell",
+        verticalAlign: "middle"
+    },
+    footer: {
+        paddingTop: "24px",
+        paddingBottom: "32px"
```

Comment:
Codacy has a fix for the issue: [Replace `········paddingBottom:·"32px"` with `····paddingBottom:·"32px",`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
    paddingBottom: "32px",
```